yi-misc-modes/src/Yi/Lexer/YAML.x

```diff
@@ -0,0 +1,52 @@
+-- -*- haskell -*-
+-- Lexer for YAML-file
+-- This is based off the syntax as described in the github manual:
+-- http://yaml.org/spec/1.2/spec.html
+-- Maintainer: Junji Hashimoto
+{
+{-# OPTIONS -w  #-}
+module Yi.Lexer.YAML ( lexer ) where
+import Yi.Lexer.Alex hiding (tokenToStyle)
+import Yi.Style
+  ( Style             ( .. )
+  , StyleName
+  )
+import qualified Yi.Style as Style
+}
+
+$space = [\ ]
+
+markdown :-
+
+<0>
+{
+    ^\%.* { c Style.importStyle }
+    ^\-\-\- { c Style.importStyle }
+    ^$space*\#.* { c Style.commentStyle }
+    ^$space*[\-] { c Style.keywordStyle }
+    ^$space*[^\:]+\: { c Style.keywordStyle }
+    \n
+        { c Style.quoteStyle }
+    .
+        { c Style.quoteStyle }
```

Comment:
I think making this defaultStyle is better, quoteStyle is really for Template Haskell or lisp style quotes. See https://github.com/yi-editor/yi/blob/master/yi-misc-modes/src/Yi/Lexer/JSON.x for a comparable lexer.

Reply:
Thank you for your review in your busy time
I'll fix the style.(I wonder if I should use defaultStyle or other styles.)